ln-gateway/src/lib.rs

```diff
@@ -104,12 +109,15 @@ impl LnGateway {
         self.federation_client
             .claim_outgoing_contract(contract_id, preimage, rng)
             .await?;
+
+        Ok(())
+    }
+
+    pub async fn await_contract_accepted(&self, contract_id: ContractId) {
         self.federation_client
             .await_claimed_outgoing_accepted(contract_id)
             .await;
         debug!("Claim transaction accepted");
-
-        Ok(())
     }
 }
```

Comment:
the previous 'pay_invoice' awaited the `self.federation_client.await_claimed_outgoing_accepted(contract_id)` bevore returning` Ok(())` pay invoice is used by ln-gateway/src/bin/ln_gateway. Can we be certain that leaving this out won't have side affects ?  (the new fn is only used by integrationtests.rs not the fn that initially used this fn)

Reply:
That is a good catch, it will affect the behavior of the gateway API to basically become non-blocking.  I don't necessarily think that is bad behavior but I will add that functionality back until we decide on whether to create another endpoint to block until the claim is accepted.